lib/tanker/identity.rb

```diff
@@ -108,11 +128,11 @@ def self.get_public_identity(serialized_identity)
 
       identity = deserialize(serialized_identity)
 
-      if identity['target'] == 'user'
-        public_keys = ['trustchain_id', 'target', 'value']
-      else
-        public_keys = ['trustchain_id', 'target', 'value', 'public_encryption_key', 'public_signature_key']
-      end
+      public_keys = if identity['target'] == 'user'
+                      ['trustchain_id', 'target', 'value']
+                    else
+                      ['trustchain_id', 'target', 'value', 'public_encryption_key', 'public_signature_key']
+                    end
```

Comment:
I prefer the previous form but meh

Reply:
Ah yes, c'est le linter qui trouve ca plus idiomatique, perso les deux me vont =)